dev/devicelab/lib/framework/ios.dart

```diff
@@ -102,6 +104,40 @@ Future<bool> containsBitcode(String pathToBinary) async {
   return !emptyBitcodeMarkerFound;
 }
 
+Future<bool> bonjourServiceFound(String appBundlePath) async =>
+  (await eval(
+    'plutil',
+    <String>[
+      '-extract',
+      'NSBonjourServices',
+      'xml1',
+      '-o',
+      '-',
+      path.join(
+        appBundlePath,
+        'Info.plist',
+      ),
+    ],
+    canFail: true,
+  )).contains('_dartobservatory._tcp');
+
+Future<bool> localNetworkUsageFound(String appBundlePath) async =>
+  await exec(
+    'plutil',
+    <String>[
+      '-extract',
+      'NSLocalNetworkUsageDescription',
+      'xml1',
+      '-o',
+      '-',
+      path.join(
+        appBundlePath,
+        'Info.plist',
+      ),
+    ],
+    canFail: true,
+  ) == 0;
```

Comment:
Don't really care what it says since that seems like a moving target, just that it exists.